examples/Deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -61,6 +61,16 @@
     "### Import required libraries and functions"
```

Comment:
<p>can we install this in the first notebook as well? because we save the user features, item features and configuration files (feature views) into the feature repo folder.. we need that folder to be initialized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
@karlhigley we also have installing `tree` issue.. i

`apt-get install tree` does not always work. It might require `apt-get update` first.. what shall we do? add both?

Reply:
I wondered if we could use a Python alternative to `tree`, but assuming people are running in the containers, we can probably do `apt-get update; apt-get install tree`?